src/components/app.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
+import logo from "../assets/200424060716-nasa-worm-logo-removebg-preview.png";
+import "../styles/app.css";
+
+const App = () => {
+  const [searchResults, setSearchResults] = useState([]);
+
+  return (
+    <div className="bg">
+      <div className="app">
+        <img className="nasa-logo" src={logo} alt="nasaLogo" />
```

Comment:
Your alt text here should be in plain english, as they are used by screen readers they dont have to be in any other format